### DISCUSSION TITLE
Can we use deepmd in halide perovskite materials?

### DISCUSSION THREAD

**Discussion by theAfish** [deepmodeling/deepmd-kit#895] [Q&A] [answered]
Hi all

I am wondering to use DPMD to investigate the growth process of low-dimensional halide perovskite (2D PSK in short). PSK could be very complex. For example, organic-inorganic hybrid perovskite such as MAPbI3 contains methylammonium (CH3NH3+) as well as inorganic lead and iodide. This will be more complex If we introduce larger organic cations (like Butylammonium(BA)) for 2D PSKs. So the number of atoms is really large (a 2x2x2 pseudo-cubic cell will contain 90+ atoms) and cannot be easily computed via AIMD. As a result, it is expensive for large-scale and long time simulation, and that's why I think DPMD could be helpful in my research.

However, I am concerned about several details:

1. The system contains different types of atoms (for some complex compositions: C, N, H, Pb, I, and even Cs, Br). Does this mean we need a relatively large dataset? If so, is the process of obtaining a dataset through DFT calculations overly consuming? and how much of the training set is suitable? (I saw that the water example contains a dataset with 1000 data points, so for complex systems, will it need a large number of data points?)
2. If we training inorganic 2D PSK CsPbI3 with BA larger cations as spacers, could we used the final potential file to run MD for pure CsPbI3 without BA cations?

**Comment by njzjz** [comment, marked as answer]
1. You may first read the [concurrent learning method](https://www.sciencedirect.com/science/article/pii/S001046552030045X) and [the papers citing that paper](https://app.dimensions.ai/discover/publication?or_subset_publication_citations=pub.1124804422). It's a way to reduce the dataset size.
2. You can do it, but I don't think it will be as accurate as the potential trained with BA cations.

**Reply by theAfish** [reply]
Thank you very much! I'll look into it.
